src/ppl/src/IOPlacer.cpp

```diff
@@ -2802,6 +2802,40 @@ void IOPlacer::initNetlist()
   }
 }
 
+void IOPlacer::commitConstraintsToDB()
+{
+  for (Constraint& constraint : constraints_) {
```

Comment:
does this handle mirroring?

Reply:
It wasn't, but my last commits are covering mirrored pins. I've tested with PPL unit tests and the values are correct.